src/Utilities/FlowAnalysis/Analysis/CopyAnalysis/CopyAnalysis.CopyDataFlowOperationVisitor.cs

```diff
@@ -325,6 +335,180 @@ protected override CopyAnalysisData GetAnalysisDataAtBlockEnd(CopyAnalysisResult
             protected override bool Equals(CopyAnalysisData value1, CopyAnalysisData value2)
                 => value1.Equals(value2);
 
+            public override (CopyAbstractValue Value, PredicateValueKind PredicateValueKind)? GetReturnValueAndPredicateKind()
+            {
+                // Filter out all the local symbol and flow capture entities from the return value.
```

Comment:
This needs to be done to ensure that analysis entities local to a method don't leak into the caller's analysis data - which also caused quite a few interprocedural asserts due to inconsistent analysis data.